libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -346,6 +344,7 @@ void SITL_State::_simulator_servos(struct sitl_input &input)
         wind_dir_z =     _sitl->wind_dir_z_active;
         
         // pass wind into simulators using different wind types via param SIM_WIND_T*.
+        const float altitude = _sitl->state.height_agl;
```

Comment:
`AP:baro().get_altitude()` will always return altitude above home.
But `_silt->state.height_agl` has the capacity to contain actual AGL, right?

So in terms of wind calculation, that's even better! Nice.